spec/puppetserver/ca/action/prune_spec.rb

```diff
@@ -95,13 +95,17 @@
   end
 
   describe 'update' do
-    it 'bumps CRL version up by 1' do
+    it 'bumps CRL number up by 1' do
       ca_key = OpenSSL::PKey::RSA.new(512)
       ca_cert = create_cert(ca_key, "Bazzup")
       ca_crl = create_crl(ca_cert, ca_key)
 
       subject.update_pruned_CRL([ca_crl], ca_key)
-      expect(ca_crl.version).to eq(2)
+
+      extensions = ca_crl.extensions.select { |ext| ext.oid == "crlNumber"}
+      extensions.each do |ext|
+        expect(ext.value).to match("1")
```

Comment:
It looks like `match` is typically used when comparing against a regex: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/match-matcher
We might just want `eq` for this?